module-4/backend/week_two.md

```diff
@@ -5,17 +5,29 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is Webpack and why is it useful?
+it is a complier, it allows all of your front end tools to be loaded into production
 2. When do you want to use event delegation?
+when you want to add synchronous(sp?) actions. ie on click, on push etc etc
 3. What's one difference between ES5 and ES6?
+the way a callback is called. function(done) vs (done) =>
 4. What's the deal with semi-colons in JavaScript?
+they signal the end of a block
 5. How are you using the MVC design pattern in your Quantified Self project?
+when we serve our API in the BE.
 6. How do you execute raw SQL in node?
+set your db to a var, then someDb.raw('INSERT SQL')
 7. What is CORS?
+http acces control
 8. What are some steps to avoid CORS?
+npm i cors
 
 #### Review  
 
 9. Why do people say "HTTP is stateless"?
+because it doesnt know what the last request was, or the next request will be.
 10. What is a RESTful API?
+where you use proper endpoints in your api
```

Comment:
a RESTful API also follow quite a few naming conventions from the route to the request type, etc.